src/actions/transferActions.js

```diff
@@ -70,7 +70,7 @@ async function _directTransfer (txRequest: {
     .toString()
   const _wallet = createWallet(fromAccount)
 
-  let txHash = await _wallet.sendTransaction({
+  let { txHash }= await _wallet.sendTransaction({
```

Comment:
This is correct.  https://github.com/ventureum/Chainsfr/blob/master/src/types/wallet.flow.js#L27

Reply:
inside promise is an object, not a string